src/main/java/net/rptools/maptool/model/TokenFootprint.java

```diff
@@ -68,6 +70,9 @@ public TokenFootPrintDto toDto() {
     dto.setId(id.toString());
     dto.setIsDefault(isDefault);
     dto.setScale(scale);
+    if (localizedName != null) {
+      dto.setLocalizedName(localizedName);
+    }
```

Comment:
The localised name probably shouldn't be transferred or saved, since it could be different on different clients